packages/manager/modules/octavia-load-balancer/src/loadbalancers/loadbalancers.controller.js

```diff
@@ -0,0 +1,40 @@
+import { TRACKING_OCTAVIA_LOAD_BALANCERS_PREFIX } from './loadbalancers.constants';
+
+export default class OctaviaLoadBalancersCtrl {
+  /* @ngInject */
+  constructor(atInternet, Alerter, $translate) {
+    this.atInternet = atInternet;
+    this.Alerter = Alerter;
+    this.$translate = $translate;
+  }
+
+  createLoadBalancer() {
```

Comment:
I think all the redirection methods can be defined in the routing and used directly in the template

Reply:
See comment https://github.com/ovh/manager/pull/10044#discussion_r1344723363

Reply:
Indeed it does not seem to be doable using the directive on the oui-button.
But tracking could be from the routing and as such leaving you only the call to the method to make on the on-click.